### ISSUE TITLE
[Fishing] Missing error message 'Your fish got away.'

### ISSUE THREAD

**Issue by ishidinho** [chromiecraft/chromiecraft#2410]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [X] Horde

### Content Phase:

- [X] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

When a fish escapes during fissing, no interface error message appears. 

### Expected Blizzlike Behaviour

When a fish escapes during fishing because of too low fishing skill a interface error message 'Your fish got away.' should be displayed.

### Steps to reproduce the problem

My test setup:
1. Fishing skill 258
2. Try to fish in Camp Mojache in Feralas
3. No error message if a fish escapes

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/fc16abc30ea2e9412627ddd9d15868eac8aabb53

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by ishidinho** [comment]
Note: I disabled all addons, of course.

**Comment by Shineslippers** [comment]
Wowpedia [Fishing](https://wowpedia.fandom.com/wiki/Fishing?oldid=2388459#Fishing_level_requirements)
Wowwiki [Patch 3.1](https://wowwiki-archive.fandom.com/wiki/Patch_3.1.0#Fishing)
> You can now fish anywhere, regardless of skill. **_Every catch has the potential for fishing skill gains_**, but you are likely to catch worthless junk in areas that are too difficult for your skill.

In your case i don't understand what you mean by saying `fish escape`. The message `You fish got away` you can get if not right click on Fishing Bobber.
![WoWScrnShot_112721_191229](https://user-images.githubusercontent.com/85256266/143688755-369f024b-302b-4284-9a9d-b5bd4bf68b8d.jpg)

I'm assuming this report about missing `worthless junk` that you can get if you fishing skill not high enough

**Comment by Azcobu** [comment]
@ishidinho - when you say a fish escaped, do you mean that an empty loot screen appeared, or did you get a junk item?

If it was junk, that would be expected behaviour: "all casts will catch something and award a chance for a skill-up. However, if your skill level is too low for a zone, you will catch mostly vendor trash items. The skill level required to guarantee "no junk" catches is equivalent to the old "no get away" level." If, on the other hand, you got an empty loot screen, then that would be an issue.

**Comment by Shineslippers** [comment]
Yes there is empty loot and skill up. I guess issue with junk. I never got any junk there.
![image](https://user-images.githubusercontent.com/85256266/143766599-3a6818ec-f7f5-4395-a37b-02ba7b97ecc0.png)

**Comment by ishidinho** [comment]
@Azcobu 

Really nothing appears. No fish. No item. No empty loot screen. No error message.

**Comment by Shineslippers** [comment]
No empty loot screen could be caused by addons like xloot.

**Comment by ishidinho** [comment]
I preproduced the issue with all addons disabled before posting the issue.

**Comment by Shineslippers** [comment]
@Azcobu same as https://github.com/azerothcore/azerothcore-wotlk/issues/9208. Sometimes no loot. There is probably because day/night schedule. Can't reproduce no `loot screen`.

**Comment by Azcobu** [comment]
Merging this ticket into https://github.com/azerothcore/azerothcore-wotlk/issues/9208 and renaming that to reflect that it's a broader problem.

**Comment by Shineslippers** [comment]
Yea, i mentioned it 2 messagesa above https://github.com/chromiecraft/chromiecraft/issues/2410#issuecomment-981116941
